CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
+## 2019-??-?? - 1.0.3
+  - Added items to satisyf puppet-lin and pdk (Thanks alexjfisher)
+  - Added better validation for verifying proper parameters passed (Thanks alexjfisher)
+  - Changed syntax on attributes used by module to satisfy REXML (Thanks MacGregor001)
```

Comment:
This changed isn't actually merged (or in this branch).

Reply:
No, not yet. I was just staging these to get ready to wrap this up to into a release to push to the forge. The changes are elsewhere and I was using this branch as a placeholder.